sage-ios/SAGE/SAGE/CheckinViewController.swift

```diff
@@ -0,0 +1,263 @@
+//
+//  CheckinViewController.swift
+//  SAGE
+//
+//  Created by Andrew on 10/3/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+
+import FontAwesomeKit
+import SwiftKeychainWrapper
+
+class CheckinViewController: UIViewController {
+
+    let locationManager = CLLocationManager()
+    let checkinView = CheckinView()
+    let defaultTitleLabel = UILabel()
+    let sessionTitleLabel = UILabel()
+    var startTime: NSTimeInterval = 0.0
+    var inSession: Bool = false
+
+    //
+    // MARK: - ViewController Lifecycle
+    //
+    override func loadView() {
+        self.view = self.checkinView
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.setupDefaultTitleLabel()
+        self.setupSessionTitleLabel()
+        self.setupBarButtonItems()
+        
+        self.checkinView.startButton.addTarget(self, action: "userPressedBeginSession:", forControlEvents: .TouchUpInside)
+        self.checkinView.endButton.addTarget(self, action: "userPressedEndSession:", forControlEvents: .TouchUpInside)
+
```

Comment:
you're never using the sender parameter for userPressedBeginSession: and userPressedEndSession: so why do the functions include them??
you could you just userPressedBeginSession